src/server/rdb_test.cc

```diff
@@ -157,9 +157,9 @@ TEST_F(RdbTest, Stream) {
   EXPECT_THAT(resp, ArrLen(2));
 
   resp = Run({"xinfo", "groups", "key:1"});  // test dereferences array of size 1
-  EXPECT_THAT(resp, ArrLen(8));
   EXPECT_THAT(resp.GetVec(), ElementsAre("name", "g2", "consumers", IntArg(0), "pending", IntArg(0),
```

Comment:
While here, you could use the newer syntax:
`EXPECT_THAT(resp, RespArray(ElementsAre(...)))`;

Reply:
nice, thanks!